pkgs/by-name/pi/pied/package.nix

```
@@ -25,11 +27,14 @@ flutter327.buildFlutterApplication rec {
     gst_all_1.gst-plugins-base
     gst_all_1.gst-plugins-good
     gst_all_1.gst-plugins-bad
+    killall
+    piper-tts
   ];
 
   postInstall = ''
     install -D flatpak/com.mikeasoft.pied.desktop -t $out/share/applications
     install -D flatpak/com.mikeasoft.pied.png -t $out/share/pixmaps
+    ln -s ${piper-tts}/bin/piper $out/bin/piper
```

there is a function to get the binary path : `lib.getExe`

also, for scripts and documentation, personnaly, i prefer long option rather than short one when available

```suggestion
    ln --symbolic ${lib.getExe piper-tts} $out/bin/piper
```

---

That definitely looks better, adding it now.